library/src/ui/index.js

```diff
@@ -1 +1,72 @@
-window.CoNDeT.ui = {}
\ No newline at end of file
+window.CoNDeT.ui = {
+    BaseComponent: {
+        state: {
+
+        },
+        common: {
+
+        },
+        init: function(common, props) {
+            this.ref = this.onInit();
+        },
+        update: function(common, props) {
+            children.update();
+            this.onUpdate();
+            this.ref.onUpdate();
+        },
```

Comment:
`this.ref` jest elementem dom. Więc wywołanie na nim `onUpdate` na 99% wywali type error  
Tutaj też do należy podać `common` i `props` do `this.onUpdate()`  
W sumie to nie wiemy, czy komponent chce aktualizować dzieci. Więc usunąłbym `children.update();`